Templates/QueueTrigger-NodeJS/index.js

```diff
@@ -1,4 +1,4 @@
 ﻿module.exports = function (context, myQueueItem) {
-    context.log('Node.js queue trigger function processed work item', myQueueItem.id);
```

Comment:
Should probably remove the "work item" nomenclature from these samples. I had that there because they used to be JSON work items with an ID.

I'd search for all "work item" (I used it in other samples) and just reword to "processed queue item"